kuma/wiki/tests/conftest.py

```diff
@@ -185,6 +185,103 @@ def doc_hierarchy(wiki_user):
 
 
 @pytest.fixture
+def doc_hierarchy_with_zones(settings, doc_hierarchy, wiki_user_2, wiki_user_3):
```

Comment:
For future discussion - when a fixture is just used in one test or test file, should it go in ``conftest.py`` or in that test file?  I'm leaning toward test-specific fixtures in the test files, but I'm willing to wait until we have more than 50% of tests converted to pytest to start moving things around.